src/util/searchHelpers.ts

```diff
@@ -33,20 +33,20 @@ const getArticleTypesField = (resourceTypes?: string[]) => {
   else return {};
 };
 
-const getContextTypes = (resourceType: string[] | undefined) => {
+const getContextTypes = (resourceType: string[] | undefined, contextTypes: string[] | undefined) => {
```

Comment:
Skulle vi lagt på en fallback-value for contextTypes? typ: `contextTypes: string[] | undefined = []`

Reply:
Det kunne vi sikkert gjort, men om contextTypes er undefined så ingoreres det vel så tipper effekten er lik.

Reply:
La inn fallback for å slippe å skrive om e2e-test